### ISSUE TITLE
Key ESC/ENTER crashes runebender

### ISSUE THREAD

**Issue by piaoger** [linebender/runebender#245]
- env: macOS Big Sur
- error message:

``` text
2021-04-20 15:33:45.815 runebender[14467:438326] +[NSView doCommandBySelector:]: unrecognized selector sent to class 0x7fff8865da08
Illegal instruction: 4
```

**Comment by cmyr** [comment]
Okay, something weird is happening on big sur in a bunch of druid applications. The crash seems very weird, though, can you reproduce that consistently? Do you need to do anything special?

**Comment by piaoger** [comment]
Yes, it's reproducible easily. You can just try to exit runebender by "ESC"  key right after it is launched.
You can also find more details from attached error log.

[apple-error-report.log](https://github.com/linebender/runebender/files/6347217/apple-error-report.log)

**Comment by cmyr** [comment]
@piaoger thanks!
